<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdDataDownloaderClass</class>
 <widget class="QWidget" name="SdDataDownloaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SdDataDownloader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="ipEdit">
       <property name="text">
        <string>192.168.2.10</string>
       </property>
       <property name="placeholderText">
        <string>ip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Begin sector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="beginSectorSpinBox">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End sector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="endSectorSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SdDataDownloader.qrc"/>
 </resources>
 <connections/>
</ui>
